src/components/side_panel/chart/radar_chart/radar_chart_design_panel.xml

```diff
@@ -0,0 +1,21 @@
+<templates>
+  <t t-name="o-spreadsheet-RadarChartDesignPanel">
+    <GeneralDesignEditor
+      figureId="props.figureId"
+      definition="props.definition"
+      updateChart="props.updateChart">
+      <t t-set-slot="general-extension">
+        <LegendComponent
+          figureId="props.figureId"
+          definition="props.definition"
+          updateChart="props.updateChart"
+        />
+      </t>
+    </GeneralDesignEditor>
+    <SeriesDesignEditor
+      figureId="props.figureId"
+      definition="props.definition"
+      updateChart="props.updateChart"
```

Comment:
actually it can be made very short ! (have to check if props validation is ok with that)
```suggestion
        <LegendComponent t-props="props"/>
      </t>
    </GeneralDesignEditor>
    <SeriesDesignEditor t-props="props"/>
```